packages/launcher/src/widget.tsx

```diff
@@ -174,8 +174,12 @@ export class Launcher extends VDomRenderer<ILauncher.IModel> {
                 icon={icon}
                 iconClass={classes(iconClass, 'jp-Icon-cover')}
                 stylesheet="launcherSection"
+                title={cat + ' Icon'}
+                aria-hidden={true}
               />
-              <h2 className="jp-Launcher-sectionTitle">{cat}</h2>
+              <h2 className="jp-Launcher-sectionTitle" title={cat + ' Title'}>
```

Comment:
What is the reasoning to add title on the icon and on the header? At first glance it does not bring much value.

Reply:
reads out title in screenreader then